### DISCUSSION TITLE
请求详细的API文档

### DISCUSSION THREAD

**Discussion by photkey** [PeterDing/BaiduPCS-Py#71] [Q&A]
没有找到详细的API文档，直接读源码有点吃力（python刚入门阶段的新手）

**Comment by PeterDing** [comment]
https://github.com/PeterDing/BaiduPCS-Py/blob/master/baidupcs_py/baidupcs/api.py 中每个方法都有说明，你可以看看。
单独的api文档目前没时间写。
